### PR TITLE
Implement auto-select of last booted system

### PR DESCRIPTION
**Pull request by abraha2d**
Take 2... Much better than before...

### CODE REVIEW

File: Settings.c [hunk 1]
```diff
@@ -52,6 +52,19 @@ ShowFastbootCallback (
...
 STATIC
 EFI_STATUS
+AutoselectLastBootCallback (
+  MENU_ENTRY* This
+)
+{
+  SettingBoolSet("ui-autoselect-last-boot", !This->ToggleEnabled);
+  This->ToggleEnabled = SettingBoolGet("ui-autoselect-last-boot");
+  InvalidateActiveMenu();
+  MainMenuUpdateUi();
```
Comment (M1cha) [inline Settings.c]: you don't need to call MainMenuUpdateUi because this toggle doesn't affect what the main menu looks like.

**Review by M1cha — CHANGES_REQUESTED** [review]

**Comment by abraha2d** [comment]
Fixed.

File: Settings.c (hunk 1 above)
Comment (abraha2d) [reply Settings.c]: Ok. Fixed.

File: Include/Internal/AndroidLocator.h [hunk 2]
```diff
@@ -66,4 +66,10 @@ AndroidLocatorGetInternalROMIconPath (
   VOID
 );
 
+UINTN
+GetMenuIdFromLastBootEntry (
```
Comment (M1cha) [inline Include/Internal/AndroidLocator.h]: rename this to AndroidLocatorGetMenuIdFromLastBootEntry

**Review by M1cha — CHANGES_REQUESTED** [review]

File: Include/Internal/AndroidLocator.h (hunk 2 above)
Comment (abraha2d) [reply Include/Internal/AndroidLocator.h]: Fixed.

**Comment by abraha2d** [comment]
Fixed.

**Pull request merged**